ui/pages/confirmations/components/confirm/pluggable-section/pluggable-section.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ReactComponentLike } from 'prop-types';
+import { useSelector } from 'react-redux';
+
+import { currentConfirmationSelector } from '../../../selectors';
+
+// Components to be plugged into confirmation page can be added to the array below
+const PluggedInSections: ReactComponentLike[] = [];
```

Comment:
Since this is ultimately a standard constant, should it be uppercase and underscores?

Reply:
Done 👍